src/s1reader/s1_reader.py

```diff
@@ -307,6 +307,33 @@ def is_eap_correction_necessary(ipf_version: version.Version) -> SimpleNamespace
 
     return eap
 
+def get_track_burst_num(track_burst_num_file: str = esa_track_burst_id_file):
+    '''Read the start / stop burst number info of each track from ESA.
+
+    Parameters:
+    -----------
+    track_burst_num_file : str
+        Path to the track burst number files.
+
+    Returns:
+    --------
+    track_burst_num : dict
+        Dictionary where each key is the track number, and each value is a list
+        of two integers for the start and stop burst number
+    '''
+
+    # read the text file to list
+    track_nums = np.loadtxt(track_burst_num_file, dtype=int, usecols=[0])
+    burst_num0s = np.loadtxt(track_burst_num_file, dtype=int, usecols=[1])
+    burst_num1s = np.loadtxt(track_burst_num_file, dtype=int, usecols=[2])
+
+    # convert lists into dict
+    track_burst_num = dict()
+    for track_num, burst_num0, burst_num1 in zip(track_nums, burst_num0s, burst_num1s):
+        track_burst_num[track_num] = [burst_num0, burst_num1]
```

Comment:
```suggestion
    track_burst_info = np.loadtxt(track_burst_num_file, dtype=int)

    # convert lists into dict
    track_burst_num = dict()
    for track_num, burst_num0, burst_num1 in track_burst_info:
        track_burst_num[track_num] = [burst_num0, burst_num1]
```
Use a single read and iterate entire array (vs `zip`d variables).